src/Metadata/Property/PropertyMetadata.php

```diff
@@ -60,7 +60,7 @@ public function getType()
      *
      * @param Type $type
      *
-     * @return self
+     * @return PropertyMetadata
```

Comment:
Actually, it's here too FWIW: https://www.phpdoc.org/docs/latest/references/phpdoc/types.html

The PSR standard is still a draft, but as far as phpDocumentor (the library) is concerned, that's how it should be. And I don't think anyone would want to deviate from that without good reason...